### ISSUE TITLE
EWHM workspaces module not working

### ISSUE THREAD

**Issue by AlecsFerra** [polybar/polybar#2255]
## Checklist

* [X] I have read the appropriate section in the [contributing
  guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
* [X] I believe this issue is a problem with polybar itself and not a misconfiguration on my part.
* [X] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that
  may have already reported this problem.
* [X] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.

## Describe the bug
The ewhm workspace icons module stopped working with the new git build on arch linux (the standard one is working) and on the last git version was working correctly

The workspaces icons don't change when a workspace is focused or has a window open in it

**Expected behavior:**
The icons should change color

**Was it working before?**
* Did you also experience this bug in an earlier version of polybar (yes/no/don't know)? no, and the the standard release is still working

## To Reproduce

```dosini
[module/ewmh2]
type = internal/xworkspaces

icon-0 = 1;1
icon-1 = 2;2
icon-3= 4;3
icon-2 = 3;4
icon-4 = 5;5
icon-5 = 6;6
icon-6 = 7;7
icon-7 = 8;8
icon-8 = 9;9
icon-9 = 10;10

label-monitor = %name%
label-monitor-foreground = ${colors.buffer}

label-active = %icon%
label-active-foreground = ${colors.nord7}
label-active-padding = 2

label-occupied = %icon%
label-occupied-foreground = ${colors.nord9}
label-occupied-padding = 2

label-empty = %icon%
label-empty-foreground = ${colors.buffer}
label-empty-padding = 2

label-urgent = %icon%
label-urgent-foreground = ${colors.urgent}
label-urgent-padding = 2

label-separator = ""
label-separator-foreground = ${colors.buffer}

[bar/debug]
modules-left = ewmh2


```

## Polybar Log

```
notice: Parsing config file: /home/alecs/.config/polybar/config
warn: No fonts specified, using fallback font "fixed"
notice: Loaded font "fixed" (name=DejaVu Sans, offset=0, file=/usr/share/fonts/TTF/DejaVuSans.ttf)
```

## Environment:
* WM: xmonad (with ewmh pathces)
* Distro: Arch Liux
* Output of `polybar -vvv`:

```
polybar 3.4.0-184-g50d8a1ba

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/sbin/c++
Compiler flags: -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```

**Comment by patrick96** [comment]
Thanks for reporting. Was able to reproduce this even on herbstluftwm, so it's not isolated to xmonad.

Running `git bisect` right now to find where we broke things.

**Comment by patrick96** [comment]
Hmm, the offending commit is 4dfbba11c4f84728b31c1b7c628283730d87da41. But I don't see any clear connection to the xworkspaces module yet.

**Comment by patrick96** [comment]
Oh, this is a major bug, it breaks *all* foreground and background settings in all modules.
What's weird is that I tested those changes with my own config and never ran into this, I must have been using the wrong executable because I am now getting this issue as well. Thanks for catching this.

I will integrate this fix directly into the 3.5.0 release: #2253
